### PR TITLE
Update openmpi-jobs.md

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

File: documentation/htc_workloads/specific_resources/openmpi-jobs.md [hunk 1]
```diff
@@ -12,54 +12,65 @@ this, as long as the core count is small (4 is known to work well, 8 and 16
 becomes more difficult due to the limited number of resources).
 
 
-To get started, first compile your code using the OpenMPI in the modules
-system. For example:
+To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use the one that is available on dockerhub. OSG has a `openmpi` container. To compile your code using the OSG provided image, load the imagae first.
```
Comment (aowen-uwmad) [inline documentation/htc_workloads/specific_resources/openmpi-jobs.md:15]:
```suggestion
To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use the one that is available on dockerhub. OSG has a `openmpi` container. To compile your code using the OSG provided image, start running the container first. 
```

**Review by aowen-uwmad — APPROVED** [review]
Tested on ap20, everything works there. Can merge as is. Maybe consider the following points:

- Compile step does not work on ap40, since `/ospool/uc-shared` does not exist there. Not sure how to resolve that.
- Could add arguments to submit file to pass compiled script name, n_procs. That way you don't ever need to edit the executable.

File: documentation/htc_workloads/specific_resources/openmpi-jobs.md (hunk 1 above)
Comment (ChristinaLK) [inline documentation/htc_workloads/specific_resources/openmpi-jobs.md:15]:
```suggestion
To get started, first compile your code using the OpenMPI container. You can create your own OpenMPI container or use one that is available on Docker Hub. The OSPool also hosts an `openmpi` container you can use. To compile your code using the OSG provided image, load the image first. 
```

File: documentation/htc_workloads/specific_resources/openmpi-jobs.md [hunk 2]
```diff
@@ -12,54 +12,65 @@ this, as long as the core count is small (4 is known to work well, 8 and 16
...
     $ mpiexec -n 4 hello
-    Hello world from process 1 of 4
-    Hello world from process 3 of 4
-    Hello world from process 0 of 4
-    Hello world from process 2 of 4
-
+    Hello world from processor ap21.uc.osg-htc.org, rank 0 out of 4 processors
+    Hello world from processor ap21.uc.osg-htc.org, rank 1 out of 4 processors
+    Hello world from processor ap21.uc.osg-htc.org, rank 2 out of 4 processors
+    Hello world from processor ap21.uc.osg-htc.org, rank 3 out of 4 processors
```
Comment (ChristinaLK) [inline documentation/htc_workloads/specific_resources/openmpi-jobs.md:46]: Add a comment about exiting from the container after you compile/test.

**Review by ChristinaLK — COMMENTED** [review]
@showmic09 See Andrew's comment and mine. Once those changes are made, should be good to go.

**Comment by rachellombardi** [comment]
@showmic09 reminder that this is open

File: documentation/htc_workloads/specific_resources/openmpi-jobs.md (hunk 2 above)
Comment (showmic09) [reply documentation/htc_workloads/specific_resources/openmpi-jobs.md:47]: Done

**Pull request merged**